<launch>

    <arg name = "dest_x" default = "5"/>
    <arg name = "dest_y" default = "5"/>
    <param name = "dest_pos_x" value="$(arg dest_x)"/>
    <param name = "dest_pos_y" value="$(arg dest_y)"/>

    <node pkg="bug0_algorithm" type="wall_follow.py" name="wall_follow" output="screen"/>
    <node pkg="bug0_algorithm" type="go_to_pos.py" name="go_to_pos" output="screen"/>
    
</launch>
